### PR TITLE
added_background

### LINKED ISSUE #28: Update background of index.html
Refer the landing page of **design-1** in **Web Designs**

Create almost the same background as you see in the picture, find the resources in design-1 folder itself. (do not create any button, links, text),i.e only the leaves, glass and bg color to be updated.

**Make sure your it occupies whole screen, and no scrolling should be there.
Do not touch the 3 internal divs leave them as it is work on outer div with id=background**

### PR DESCRIPTION
**Pull request by abhishekgupta368**
Update background of index.html  fixes #28

file added:
-> modified index.css

feature implemented
-> added background

### CODE REVIEW

**Review by rajats98 — CHANGES_REQUESTED** [review]
Hey, something is broken 
1. May be you are trying to give absolute address as on your PC, give relative address
1. Try to make background of landing page not form-page, you may need to download the images of leaf and grass from the resources link, https://www.figma.com/file/jzKWEfgE8D8I8XgRG8A1hy/Landing-Page?node-id=0%3A1 
1. you should try to make its background almost same (not exactly same) as 
Landing Page

**Comment by abhishekgupta368** [comment]
Ok, sorry

**Comment by abhishekgupta368** [comment]
Where are the logo? It is not in folder

**Comment by rajats98** [comment]
> Where are the logo? It is not in folder

it's in logo-designs folder, do one more thing make a copy of logodesign1.png and name the copy as current-logo.png (in logo-designs folder itself)

use that as your logo(current-logo.png)

File: src/assets/index.css [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+    margin:0;
+}
+
+html{
+    height:100%;
+    width:100%;
+}
+
+body{
+    background-color:#CDF5E7;
```
Comment (rajats98) [inline src/assets/index.css]: background-color:#CDF5E7!important;

File: src/index.html [hunk 2]
```diff
@@ -7,15 +7,25 @@
...
-    <div id="landing_container"></div>
-    <div id="input_container"></div>
-    <div id="result_container"></div>
-  </div>
+  <body>
 
-<script src="./assets/index.js" charset="utf-8"></script>
-</body>
+    <div class="icon-class">
+      <img src="img/crop_icon.png" alt="image not found" height="100px" width="100px">
```
Comment (rajats98) [inline src/index.html]: write image as  src="./img/corp_icon.png" 
do same for all the images
and use the following images
[Landing Page (1).zip](https://github.com/CropAi/Frontend/files/4422312/Landing.Page.1.zip)

**Review by rajats98 — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (rajats98) [reply src/index.html]: @abhishekgupta368 please download this zip file for images and update, 
https://github.com/CropAi/Frontend/files/4422312/Landing.Page.1.zip
and use exact name of image (case-sensitive).[currently your images are like leaf.PNG and you are using pic1.png in img tag ]

**Review by rajats98 — CHANGES_REQUESTED** [review]
@abhishekgupta368 please download this zip file for images and update,
https://github.com/CropAi/Frontend/files/4422312/Landing.Page.1.zip
and use exact name of image (case-sensitive).[currently your images are like leaf.PNG and you are using pic1.png in img tag ]

**Comment by rajats98** [comment]
great work!

**Pull request merged**